taxi/views.py

```diff
@@ -1,3 +1,18 @@
 from django.shortcuts import render
+from django.http import HttpRequest, HttpResponse
+from taxi.models import Driver, Car, Manufacturer
+
 
 # Create your views here.
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
+
+    context = {
+        "num_cars": num_cars,
+        "num_drivers": num_drivers,
+        "num_manufacturers": num_manufacturers,
+    }
+
+    return render(request, "taxi/index.html", context)
```

Comment:
It seems like you're sending a lot of data to the template. Are you sure you need all these counts in your index view? If not, consider removing unnecessary data to improve performance. Also, if you are going to use these counts in multiple views, consider creating a utility function to avoid code duplication.

Reply:
Yes, i need to send this data